### PR TITLE
Claims Update (checked with JSONLint)

### PR DESCRIPTION
**Pull request by smaldragon**
* Updated Lambat
* Updated Monterrey
* Added Sealand
* Added Raptopia
* Updated Kallos

Added Declutter Fields to the following claims that lacked them (reasoning based on land size):
* Gempire (3)
* Krowatia (3)
* Nro'Meah (2)
* Pilltover (3)
* New Israel (Nro'Meagh) (3)

### CODE REVIEW

File: land_claims.civmap.json [hunk 1]
```diff
@@ -132,44 +132,46 @@
...
...3],[-8342,-530],[-8406,-601],[-8445,-635],[-8407,-680],[-8337,-698],[-8298,-685],[-8268,-747],[-8323,-804],[-8298,-897],[-8205,-973],[-8116,-1069],[-8057,-1108],[-8006,-1156],[-7968,-1195],[-7881,-1284],[-7837,-1300],[-7838,-1329],[-7808,-1332],[-7690,-1304],[-7616,-1290],[-7589,-1243],[-7380,-1077],[-7336,-1105],[-7295,-1087],[-7258,-1147],[-7212,-1080],[-7151,-1107],[-7093,-1152],[-7058,-1282]],[[-6320,-408],[-6350,-416],[-6374,-444],[-6406,-450],[-6460,-462],[-6488,-442],[-6512,-418],[-6520,-382],[-6520,-356],[-6486,-334],[-6462,-304],[-6440,-278],[-6406,-256],[-6390,-226],[-6358,-220],[-6350,-260],[-6366,-294],[-6356,-310],[-6334,-306],[-6304,-340]],[[-6246,-100],[-6254,-49],[-6245,-2],[-6220,15],[-6224,34],[-6207,49],[-6205,68],[-6232,88],[-6256,124],[-6291,117],[-6288,82],[-6304,56],[-6332,66],[-6361,53],[-6377,5],[-6382,-22],[-6335,-30],[-6302,-72],[-6316,-105],[-6296,-114],[-6271,-113]],[[-8588,1205],[-8580,1141],[-8656,1053],[-8708,1025],[-8744,1045],[-8744,1077],[-8828,1153],[-8880,1221],[-8920,1285],[-8896,1361],[-8828,1389],[-8780,1369],[-8728,1413],[-8664,1409],[-8584,1429],[-8524,1341],[-8504,1269],[-8532,1253]]],"collectionId":"dd432eea-e17b-471b-8b78-f22904e9ce8c"},
```
Comment (RedDevel2) [inline land_claims.civmap.json:135]: Has extra Id at the end.

File: land_claims.civmap.json [hunk 2]
```diff
@@ -132,44 +132,46 @@
...
...[6971,-911],[6967,-909],[6972,-904],[6965,-899],[6956,-898],[6951,-904],[6948,-913],[6940,-919],[6934,-919],[6933,-912],[6921,-915],[6922,-924],[6924,-941],[6915,-945],[6909,-944],[6894,-926],[6885,-916],[6883,-913],[6875,-903],[6869,-899],[6862,-898],[6856,-898],[6858,-890],[6858,-883],[6852,-880],[6853,-876],[6857,-875],[6863,-880],[6868,-879],[6874,-869],[6871,-853],[6839,-845],[6815,-840],[6692,-993],[6686,-981],[6668,-980],[6659,-986],[6649,-988],[6630,-986],[6630,-998],[6634,-1005],[6628,-1021],[6633,-1034],[6635,-1049],[6644,-1055],[6632,-1075],[6628,-1092],[6502,-1317],[6471,-1334],[6466,-1318],[6441,-1293],[6422,-1280],[6416,-1310],[6394,-1328],[6372,-1351],[6348,-1371],[6337,-1382],[6331,-1469],[6319,-1481],[6288,-1498],[6276,-1513],[6257,-1544],[6242,-1544],[6211,-1559],[6193,-1568],[6177,-1590],[6166,-1645],[6147,-1651],[6144,-1658],[6125,-1643],[6121,-1670],[6133,-1684],[6129,-1688],[6112,-1697],[6097,-1713],[6096,-1723],[6106,-1728],[6118,-1741],[6130,-1755],[6166,-1793],[6184,-1800],[6195,-1801],[6199,-1805],[6199,-1813],[6211,-1814],[6227,-1818],[6243,-1813],[6324,-1886],[6341,-1927],[6383,-1986],[6838,-1980]]],"collectionId":"dd432eea-e17b-471b-8b78-f22904e9ce8c"},
```
Comment (RedDevel2) [inline land_claims.civmap.json:138]: Has extra Id at the end.

File: land_claims.civmap.json [hunk 3]
```diff
@@ -132,44 +132,46 @@
...
...,[7465,3639],[7453,3581],[7404,3580],[7360,3635],[7306,3649],[7192,3594],[7155,3620],[7146,3656],[7094,3710],[7150,3779],[7094,3800],[7092,3829],[7039,3893],[6983,3946],[6907,3980],[6893,4036],[6916,4054],[6915,4097],[6933,4120],[6941,4169],[6903,4207],[6897,4251],[6850,4329],[6857,4384],[6801,4403],[6808,4478],[6864,4503],[6871,4565],[6839,4632],[6864,4667],[6945,4669],[6957,4717],[6924,4758],[6868,4880],[6819,4932],[6882,4995],[6861,5065],[6798,5082],[6793,5142],[6868,5199],[6929,5164],[6963,5180],[6989,5140],[7017,5098],[7043,5065],[7088,5050],[7116,5020],[7153,5012],[7190,5043],[7204,5073],[7245,5089],[7269,5199],[7324,5229],[7360,5205],[7400,5229],[7417,5295],[7472,5344],[7494,5341],[7545,5281],[7576,5274],[7606,5301],[7644,5302],[7661,5232],[7761,5244],[7772,5171],[7726,5152],[7816,5135],[7808,5055],[7880,5007],[7939,4997],[7960,4937],[7986,4971],[8034,4931],[8054,4865],[8193,4802],[8194,4744],[8219,4431],[8346,4372],[8340,4339],[8260,4303]],[[6540,4821],[6501,4861],[6488,4852],[6410,4833],[6357,4809],[6267,4841],[6217,4929],[6203,4989],[6237,5031],[6331,5037],[6429,5033],[6437,4999],[6585,4991],[6611,4891],[6556,4823]]],"collectionId":"dd432eea-e17b-471b-8b78-f22904e9ce8c"},
```
Comment (RedDevel2) [inline land_claims.civmap.json:144]: Has extra Id at the end.

**Review by RedDevel2 — CHANGES_REQUESTED** [review]
What are all these "collectionId":"civmap:collection/user" even, Others don't have them and they don't seem to serve a purpose either.

**Comment by smaldragon** [comment]
Am not sure what their purpose is either, some claims have "collectionId":"civmap:collection/user", others have a random collection-id and others still have no collection-id at all. 

It seems to not have any effect on the way claims render, so perhaps the field can be removed from all claims.

**Pull request merged**